src/sagemaker/jumpstart/curated_hub/utils.py

```diff
@@ -164,3 +179,97 @@ def create_hub_bucket_if_it_does_not_exist(
     )
 
     return bucket_name
+
+def tag_hub_content(hub_content_arn: str, tags: List[Tag], session: Session) -> List[Dict[str, List[Dict[str, str]]]]:
+    responses = []          
+    for tag in tags:
+      responses.add(session.add_tags(
+        ResourceArn=hub_content_arn,
+        Tags=[
+            {
+                'Key': tag.key,
+                'Value': tag.value
+            },
+        ]
+      ))
+
+    return responses
+    
+def find_jumpstart_tags_for_model(hub_name: str, hub_content_name: str, region: str, session: Session) -> List[Tag]:
+    list_versions_response = session.list_hub_content_versions(
+        hub_name=hub_name,
+        hub_content_type='Model',
+        hub_content_name=hub_content_name
+    )
+    hub_content_versions = list_versions_response["HubContentSummaries"]
+
+    tag_name_to_versions_map: Dict[CuratedHubTagName, List[str]] = {}
```

Comment:
nit- rename to `unsupported_hub_content_versions_map`

Reply:
Same as above

Reply:
I still disagree here Chris. If you make everything super generalized it loses it's context and makes it really difficult for people to read and understand. Have you heard of any use cases where we want to add additional tags?